certbot-nginx/certbot_nginx/parser.py

```diff
@@ -312,6 +313,32 @@ def add_server_directives(self, vhost, directives, replace):
         except errors.MisconfigurationError as err:
             raise errors.MisconfigurationError("Problem in %s: %s" % (filename, str(err)))
 
+    def create_new_vhost_from_default(self, vhost_template):
+        """Duplicate the default vhost in the configuration files.
+
+        :param :class:`~certbot_nginx.obj.VirtualHost` vhost_template: The vhost
+            whose information we copy
+
+        :returns: A vhost object for the newly created vhost
+        :rtype: :class:`~certbot_nginx.obj.VirtualHost`
+        """
+        # TODO: do the right thing for debian
```

Comment:
It means "when copying from default, put it in a new file under `sites-enabled` and symlink to that from `sites-available`, if that's the setup that we have and `sites-available/*` is imported in `nginx.conf`. Plus any other details that aren't immediately obvious."

I can say that inline, or make an issue and link to that in the comment.

Reply:
> I can say that inline, or make an issue and link to that in the comment.

I think making an issue and linking it would be great!

In general, I think we may want to think about always creating a new file regardless of the distro in the future. I think server blocks are often organized in files by themselves and this prevents us from modifying the file containing the default vhost which may attempt to be autoupdated by the OS if Nginx ships new defaults.

Reply:
The reason I didn't do it here is because that's almost true. Two server blocks for the same logical server are often included together though. Like, the `listen 80` and `listen 443` blocks would be in the same file, which is what our will end up looking like. It's when you have to create a new one based on a default that it makes most sense to be in a new server block.

Reply:
I think that keeping the HTTP and HTTPS versions in the same file is fine. I've seen both that and separate files and it really doesn't matter so let's just pick one and stick to it.

I also agree it makes sense when creating a new server block based on the default to create a new file so while we should probably update this function in the future, I'm fine with not doing it in this PR.